AjobK/Athena
# 269 followers to friends

closes #269 

<br />

 **What is it supposed to do**  
The follow/unfollow button changes to follow back/unfriend if that user follows the logged in user. A _followsYou_ field was added to the return value of the GET profile route.
![image](https://user-images.githubusercontent.com/14930277/133925917-258e0c0e-eed4-46e0-a728-eb42aed2748e.png)

<br />

 **How can we test your branch**  
Log in with two different accounts and let them follow each other to see if the text in the follow button changes properly.

<br />

 **Additional notes**  
n/a

<br />

**1. General checklist**

<br />

**2. If relevant, front-end checklist**

<br />

**3. If relevant, back-end checklist**

<br />

**4. If relevant, test these browsers**
